hudi-common/src/main/java/org/apache/hudi/common/table/timeline/HoodieArchivedTimeline.java

```diff
@@ -173,8 +174,11 @@ private String getMetadataKey(String action) {
         return "hoodieSavePointMetadata";
       case HoodieTimeline.COMPACTION_ACTION:
         return "hoodieCompactionPlan";
+      case HoodieTimeline.REPLACE_COMMIT_ACTION:
+        return "hoodieReplaceCommitMetadata";
```

Comment:
Possible to tweak existing tests to consider the clustering commit?

Reply:
yeah adding an `insert_overwrite` operation would also generate a replacecommit. don't even need clustering